### ISSUE TITLE
Firefox Issue

### ISSUE THREAD

**Issue by PleasedAlien** [AlyoshaVasilieva/luminous-ttv-ext#3]
I'm hosting the luminous server on a Raspberry Pi on my local network. The luminous extension is set to the same address in both chrome and Firefox.
It works as expected in Chrome, but in Firefox the stream fails to load and nothing shows up in the server console. 

hosting the server locally or using `https://eu.luminous.dev ` works in Firefox.

I'm using Firefox Developer Edition.

**Comment by AlyoshaVasilieva** [comment]
I don't see the same issue as you* but that specific scenario does appear to be broken. Unfortunately I have no idea why this happens, and I don't think it's necessarily in my code - TTV LOL Pro seems to have the same issue. I'm seeing a CORS error for some reason; I previously worked around [another CORS issue](https://github.com/AlyoshaVasilieva/luminous-ttv-ext/blob/5d03c0a7af119f0808863334fc84fd3692d912d9/src/ts/background.ts#L56) that only appeared in Firefox.

In short, I don't know how to fix this and I'm too lazy to spend time on it when I don't watch Twitch in Firefox. (I'm pretty sure this used to work and I don't think I've made any major changes since then.)

\* Unless you weren't running with `--debug` you should be seeing hits to the server

**Comment by ghost** [comment]
What about accepting risks and just completely disable Cors using a extension edit:or https://addons.mozilla.org/nl/firefox/addon/cors-everywhere/
